src/styled.js

```diff
@@ -14,7 +14,8 @@ export const setPragma = val => (h = val);
  * @param {String} tag DOM tagName
  * @return {Function}
  */
-export const styled = tag => (str, ...defs) => props => {
+export const styled = tag => (str, ...defs) => {  
```

Comment:
Awesome stuff @michael-klein!

I think we can shave some more bytes! 🎉 Spread operator is really compiled down to a `for` loop, and I just did a quick test, and we can get down to `919B`!

```js
export const styled = tag => function() {
  const args = Array.from(arguments);  
  const processStyles = props => {
    const className = getClassNameForCss(getCss(args.shift(), args, props));
```

Reply:
Actually, just tried it and it breaks the `args` values with `.shift()`. But using, `[0]` and a `.slice(1)` does the trick.

```js
export const styled = tag => function() {
  const args = Array.from(arguments);  
  const processStyles = props => {
    const className = getClassNameForCss(getCss(args.shift(), args, props));
```